src/nrnoc/fadvance.cpp

```diff
@@ -645,6 +645,30 @@ void nrn_calc_fast_imem(NrnThread* _nt) {
     }
 }
 
+void nrn_calc_fast_imem_fixedstep_init(NrnThread* _nt) {
+  // At end of treeset.cpp nrn_rhs(), called near end of nrn_finitialize,
+  // _nrn_sav_rhs is ionic_current and RHS is axial + ionic + stim currents
+  // So difference, scaled by area, is i_membrane_ in nA.
```

Comment:
Either this explaination or the computation is missing a '-' since you are talking about difference and in the lines after we are adding the terms. The RHS is `-(VEC_B[i] + VEC_A[i] ) * dv `. 

Bonus question:
since rhs should be equal to lhs, why capacitance current is not present in rhs? Is it because it is the first time step?

Reply:
Thanks. Tried to make the comment less ambiguous. Let me know if I succeeded.